tests/unit/moduleapi/hooks.tcl

```diff
@@ -291,6 +291,16 @@ tags "modules" {
             }
             # get the replica stdout, to be used by the next test
             set replica_stdout [srv 0 stdout]
+
+            # Manually kill the replica to avoid potential log interference
+            # from forced process termination in slow environments, which could
+            # cause the subsequent log checking test to fail.
+            exec kill [srv pid]
+            wait_for_condition 100 50 {
+                ! [is_alive [srv pid]]
+            } else {
+                fail "Replica server process didn't terminate"
+            }
```

Comment:
@oranagra please take a look this fix, a better way is to increase the wait num for ASAN, just like Valgrind.
However, I don't want to add a parameter to this minor fix, so I can only manually turn off replia in advance.
https://github.com/redis/redis/blob/ecd5e639ed4d35f71c3e67889879aee0e8fb3919/tests/support/server.tcl#L100-L105

Reply:
without a large RDB, this slowness occurred with ASAN.
I used `taskset -c 0` locally to reproduce it, and it occasionally took 10 secs and can be closed.
But I realized that I only gave a 5 seconds threshold for manually stopping, but it was 10 seconds for kill_server. However, manually closing it was effective. I need to confirm why.